optimize/self-managed/optimize-deployment/migration-update/camunda-7/3.13-to-3.14.md

```diff
@@ -17,6 +17,11 @@ Here you will find information about:
 - Changes in behavior (for example, due to a new feature)
 - Changes in translation resources
 
+## Versioning
+
+From Optimize 3.14 onwards, instances of Optimize running with Camunda Platform 7 will exclusively be using the 3.x.x versioning scheme. Conversely, instances of Optimize running with Camunda Platform 8 will exclusively be using the 8.x.x versioning scheme. This means you will only be able to update to Optimize 3.14 if you are currently using Optimize 3.13 or below. Optimize instances of versions 8.5 and below will not be upgradeable to Optimize 3.14.
```

Comment:
> if you are currently using Optimize 3.13 or below

This is confusing, because customers can't update from 3.12 -> 3.14, for example

Reply:
Can you elaborate, how will customers get to 3.14 if they cant update? Our [docs](https://docs.camunda.io/optimize/next/self-managed/optimize-deployment/migration-update/instructions/) state that we can upgrade from 3.12, we even have 3.12 > 3.13 update notes and are adding 3.13 > 3.14 note with this PR. I can also see the required upgrade plans in the repo, I dont think I'm aware of what change happened to prevent upgrading from 3.12.

Reply:
Sorry, to clarify - they can upgrade from 3.12, but not directly to 3.14, they would have to go to 3.13 first. The statement `This means you will only be able to update to Optimize 3.14 if you are currently using Optimize 3.13 or below`, specifically `or below` suggests to me that a direct migration might be possible

Reply:
Ah sorry, I thought thats how our upgrades always worked (eg getting from 1 to 3 by going 1,2 then 3). Will remove the "or below" to be less confusing.